content/tutorial/tutorial.md

```diff
@@ -611,7 +611,7 @@ Each time a player moves, `xIsNext` (a boolean) will be flipped to determine whi
 ```javascript{3,6}
   handleClick(i) {
     const squares = this.state.squares.slice();
-    squares[i] = this.state.xIsNext ? 'X' : 'O';
+    squares[i] = this.state.xIsNext ? 'O' : 'X';
```

Comment:
The `xIsNext` variable points the next player, but is switched only **after** the move is done
(see line 617: `xIsNext: !this.state.xIsNext,`).
So, we have next player being "X", -> move is done, -> "X" is marked, -> next player is switched to "O" -> and loop.

I compared this implementation to the latest of what is in docs, and see the same mechanism being implemented, and checking the example running I also verify it works as expected.